### ISSUE TITLE
Statusline shown in screenshot

### ISSUE THREAD

**Issue by jcharaoui** [molok/vim-smartusline#10]
It would be great if the README had the code to get the statusline exactly as shown in the screenshot.

**Comment by trusktr** [comment]
Yeah, it's like false advertisement. :D It'd be nice to have a default.
